composer.json

```diff
@@ -108,6 +108,9 @@
       "web/themes/contrib/{$name}": ["type:drupal-theme"],
       "drush/contrib/{$name}": ["type:drupal-drush"]
     },
+    "drupal-core-vendor-hardening": {
+      "vendor/package": ["test", "documentation"]
```

Comment:
You should add real folders to clean, see https://github.com/drupal/core-vendor-hardening/blob/8.8.x/README.txt
If there aren't any you want to specify here then please remove this extra config